src/staticsampler.jl

```diff
@@ -65,7 +65,8 @@ function Nested(ndims,
         proposal = if ndims < 10
             Proposals.Uniform()
         else#if 10 ≤ ndims ≤ 20
-            Proposals.RWalk()
+            Proposals.RWalk() # _Proposals.RStagger()_ can be taken as an alternative to _Proposals.RWalk_ 
+            # for ndims > 20, _Proposals.HSlice_ method if _gradient_ is provided, else _Proposals.Slice_ method to be used (this will be included later on)  
```

Comment:
the comments here are a bit distracting, and like my point above the `RStagger` is an alternative, but this code isn't ever going to choose `RStagger`, so it doesn't make as much sense to talk about it here.

The comments about the future work can be helpful, but I would keep them in a separate PR. This way I can make a release of the code as soon as this PR is merged!

Reply:
alright, removing the future work comments from here. shall include them in the respective feature PR